docs/docs/submit-to-creator-showcase.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Submit to Creator Showcase
+---
+
+Want to be a part of the [Creator Showcase](link TBA)? Follow these instructions.
+
+## Steps
+
+There are only two major steps :)
+
+1.  If this is your first contribution to the Gatsby open source repo, follow the [Contribution guidelines](https://next.gatsbyjs.org/docs/how-to-contribute/#contributing-to-the-repo) to get push access rights.
```

Comment:
"to get push access rights" might be misleading here, as people would create pull request and not pushing to our repository.

I think common steps could be copied from https://github.com/gatsbyjs/gatsby/blob/f19287ec91505501cd1332345c4f0a414fabded6/docs/docs/site-showcase-submissions.md and change "edit `sites.yml`" step to edit `creators.yml` from below.